src/ConsoleContactAppManager/Program1.cs

```diff
@@ -0,0 +1,25 @@
+﻿// <copyright file="Program1.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace ContactManagerApp
+{
+    /// <summary>
+    ///  class program has the enum for options to be used for the switch case in main class.
+    /// </summary>
+    internal partial class Program
```

Comment:
Program class is not needed move enum outside

Reply:
Enums grouped in a file without class.